package.json

```diff
@@ -204,7 +203,10 @@
 			"wp-scripts lint-js"
 		],
 		"{docs/{toc.json,tool/*.js},packages/{*/README.md,*/src/{actions,selectors}.js,components/src/*/**/README.md}}": [
-			"npm run docs:build"
+			"node ./docs/tool"
+		],
+		"packages/**/*.js": [
```

Comment:
I wanted to ignore files related to tests, but it looks like I can't do that (`ignore` [option](https://www.npmjs.com/package/lint-staged#options) affects all tasks).

Reply:
Also note that although the script updates some `README` file, the exit code will be `0` so it won't prevent the commit from happening. The `docgen` package doesn't have any knowledge of whether the output file has been modified with regards to a previous version, so if we strongly wanted to have that it should be done in a further PR implementing some kind of git command.